dpc-impl/app/models/user.rb

```diff
@@ -69,7 +68,7 @@ def provider_orgs
     api_client = ApiClient.new
     api_client.get_provider_orgs(self.implementer_id)
     if api_client.response_successful?
-      api_client.response_body.sort_by{|e| e[:created_at]}.reverse
+      api_client.response_body.sort_by{|e| e["created_at"]}.reverse
```

Comment:
Shall we use single quotes as above?
```suggestion
      api_client.response_body.sort_by{|e| e['created_at']}.reverse
```